presets/4.3/tune/UAV_tech_5in.txt

```diff
@@ -0,0 +1,48 @@
+#$ TITLE: UAV Tech - 5in (~630g AUW)
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: OFFICIAL
+#$ KEYWORDS: 5in, 5", freestyle
+#$ AUTHOR: UAV Tech (Mark Spatz)
+#$ DESCRIPTION: Mark is a Betaflight contributor, youtube content creator, and professional tuner (www.theuavtech.com)
```

Comment:
this is optional, but you can use:
`#$ PARSER: MARKED`
and then can at least insert links people can click on, for example:
`[www.theuavtech.com](https://www.theuavtech.com)`